taskfile.yaml

```diff
@@ -545,7 +543,8 @@ tasks:
       - "{{.ROOT_DIR}}/requirements.txt"
```

Comment:
We need to remove the "{{.PACKAGE}}/**/*" line.

Reply:
sorry this is a regression after merging main into branch